src/solver/variable/surveyresults/surveyresults.cpp

```diff
@@ -139,7 +139,7 @@ void InternalExportDigestLinksMatrix(const Data::Study& study,
     }
 }
 
-void ExportGridInfosAreas(const Data::Study& study, const String& folder)
+static void ExportGridInfosAreas(const Data::Study& study, IResultWriter::Ptr writer)
 {
     Clob out;
```

Comment:
We should try to remove **Yuni::Clob** from this function, and from this file as well (if it's easy to do and test), and replace it with **std::string**, faster at execution

Reply:
Sure